android/src/main/java/com/twiliovoicereactnative/VoiceApplicationProxy.java

```diff
@@ -35,6 +35,7 @@ public void onServiceDisconnected(ComponentName name) {
       voiceServiceApi = null;
```

Comment:
The only change in this file is additional new lines, could we just revert the changes?

Reply:
ahh no, just nonsense.. not sure how they got introduced.. thanks!